products/idn/tools/cli/index.md

```diff
@@ -0,0 +1,210 @@
+---
+id: cli
+title: CLI
+pagination_label: CLI
+sidebar_label: CLI
+sidebar_position: 1
+sidebar_class_name: cli
+keywords: ['cli']
+description: Learn how to use the SailPoint CLI in this guide. 
+slug: /tools/cli
+tags: ['CLI']
+---
+
+## Start using the CLI
+
+Learn how to use the SailPoint command line interface (CLI) in this guide. You can use the CLI to interact with IDN and the rest of SailPoint's SaaS platform in a programmatic way. You can use the CLI to perform many functions you would have otherwise used Postman or custom scripts to perform before, and you can perform them directly on the command line with minimal setup. 
+
+![Sail](./assets/img/vhs/Sail.gif)
+
+## Contents
+
+- [Requirements](#requirements)
+- [Installation](#installation)
+  - [Homebrew](#homebrew)
+  - [Manual Installation](#manual-installation)
+- [Configuration](#configuration)
+  - [Assisted configuration](#assisted-configuration)
+  - [Manual configuration](#manual-configuration)
+    - [Environment variable configuration](#environment-variable-configuration)
+  - [Usage](#usage)
+  - [GitHub](#github)
+  - [Contribution](#contribution)
+
+## Requirements
+
+- Golang version 1.18 or above. You can download it [here](https://go.dev/dl/). You can run `go version` to check your version. 
+
+## Installation
+
+There are two ways to install the SailPoint CLI: you can use a package manager for the OS of your choice, or you can manually install it.
+
+Prebuilt binaries for OS X, Windows, and Linux are provided in [Releases](https://github.com/sailpoint-oss/sailpoint-cli/releases).
+
+### Homebrew
+
+MacOS users can use [Homebrew](https://brew.sh/) to install the CLI. Run these commands in your terminal:
+
+```sh
+brew tap sailpoint-oss/tap && brew install sailpoint-cli
+```
+
+Then make sure you can run the `sail` command.
+
+### Manual Installation
```

Comment:
I would strongly prefer to remove the manual build and install from the docs for now and keep them simple. A large majority of our developer community are not actual developers, and I want to keep things simple for them to start--especially at launch. We can always add the manual installation docs later.

For now, let's just focus on the simplest path for them to get the CLI through their OS.